test-cases/longevity/longevity-parallel-schema-changes-12h.yaml

```diff
@@ -1,16 +1,15 @@
 test_duration: 800
 
-prepare_write_cmd:  ["cassandra-stress write cl=QUORUM n=20971520 -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=80 -pop seq=1..20971520 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5",
-                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=100 -clustering-row-count=5000                       -clustering-row-size=uniform:1024..4096 -concurrency=10 -connection-count=10 -consistency-level=quorum -rows-per-request=10 -timeout=30s -retry-number=30 -retry-interval=80ms,1s",
-                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=100 -clustering-row-count=5000 -partition-offset=101 -clustering-row-size=uniform:4096..8192 -concurrency=10 -connection-count=10 -consistency-level=quorum -rows-per-request=10 -timeout=30s -retry-number=30 -retry-interval=80ms,1s"
+prepare_write_cmd:  [
+                     "cassandra-stress write cl=QUORUM n=20971520 -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=80  -pop seq=1..20971521 -col 'n=FIXED(10) size=FIXED(128)' -log interval=10",
```

Comment:
The `-port jmx=6868` is redundant.
See https://github.com/scylladb/scylla-cluster-tests/pull/4681

The same below.

Reply:
Why "n=20971520" and " -pop seq=1..20971521" ?

Reply:
removed. Fixed number of seq.